### PR TITLE
Remove-DbaAgentOperator to support pipeline from Get-* function 

### PR DESCRIPTION
**Pull request by MikeyBronowski**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes # )
 - [] Breaking change (effects multiple commands or functionality, fixes # )
 - [x] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [x] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### Purpose

To support pipeline from Get-* function, as it was not working properly. 

### Approach

Similar to #7997 

### Commands to test

`Get-DbaAgentOperator -SqlInstance SRV1 | Out-GridView -Title 'Select SQL Agent operator(s) to drop' -OutputMode Multiple | Remove-DbaAgentOperator`

### CODE REVIEW

File: functions/Remove-DbaAgentOperator.ps1 [hunk 1]
```diff
@@ -49,42 +55,65 @@ function Remove-DbaAgentOperator {
 
         This removes an operator named DBA from the instance.
 
+    .EXAMPLE
+        PS C:\>  Get-DbaAgentOperator -SqlInstance SRV1 | Out-GridView -Title 'Select SQL Agent operator(s) to drop' -OutputMode Multiple | Remove-DbaAgentOperator
+
+        Using a pipeline this command gets all SQL Agent operator(s) on SRV1, lets the user select those to remove and then removes the selected SQL Agent alert category(-ies).
+
     #>
-    [CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess, ConfirmImpact = "High")]
```
Comment (potatoqualitee) [inline functions/Remove-DbaAgentOperator.ps1:53]: aw was there an issue with the defaultparametersetname?

Comment (MikeyBronowski) [reply functions/Remove-DbaAgentOperator.ps1:53]: Not I am aware of, I re-used the template for other commands, so looks like this part is missing from all other commands that were recently created.

Comment (potatoqualitee) [reply functions/Remove-DbaAgentOperator.ps1:53]: Ah k. So if you can add back the defaultparametersetname then remove the "non-pipeline" parameterset, it'll be ready to merge 👍🏼

**Comment by potatoqualitee** [comment]
will merge once tests pass then publish the module again -- i'd forgotten to commit my changes to the psd1 and psm1! should be fixed within the hour

**Pull request merged**